### PR TITLE
Dev-6043 Add Lighthouse CI

### PR DESCRIPTION
**Pull request by jonhill13**
**Do Not Merge**

- [x] Ross and/or Beth's approval.

- [x] LHCI Token

**High level description:**

Adds Lighthouse CI to our build process.

**Technical details:**

> • Adds steps to travis to install chrome and lighthouse on the server and then build productions assets and run lighthouse after linter and tests run.

> • Adds lighthouse config file for urls to test and code location.

**JIRA Ticket:**
[DEV-6043](https://federal-spending-transparency.atlassian.net/browse/DEV-6043)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket

Reviewer(s):
- [ ] Code review complete

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
         "errors": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" --quiet; exit 0;",
         "dev": "npx babel-node --no-babelrc src/js/scripts/buildSitemap.js --only 'src/js/scripts/**' && webpack --progress --config ./webpack/webpack.dev.config.js",
         "prod": "npx babel-node --no-babelrc src/js/scripts/buildSitemap.js --only 'src/js/scripts/**' && webpack --progress --config ./webpack/webpack.prod.config.js",
-        "travis": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" && ./node_modules/.bin/jest --runInBand;"
+        "travis": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" && ./node_modules/.bin/jest --runInBand;",
+        "lh": "npx babel-node --no-babelrc src/js/scripts/buildSitemap.js --only 'src/js/scripts/**' && webpack --progress --config ./webpack/webpack.prod.config.js && lhci autorun"
```
Comment (MaxwellKendall) [inline package.json:16]: Do w e need to build the sitemap here?

File: src/lighthouserc.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    ci: {
+        upload: {
+            target: 'temporary-public-storage'
+        },
+        collect: {
+            staticDistDir: './public', // location of static files for lighthouse CI to serve
```
Comment (MaxwellKendall) [inline src/lighthouserc.js:7]: noice

**Review by MaxwellKendall — COMMENTED** [review]
Just some comments. Does this work?

File: lighthouserc.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    ci: {
+        upload: {
+            target: 'temporary-public-storage'
+        },
+        collect: {
+            staticDistDir: './public', // location of static files for lighthouse CI to serve
```
Comment (MaxwellKendall) [inline lighthouserc.js:7]: not sure we actually have a static site in the way it expects us to?

Comment (jonhill13) [reply lighthouserc.js:7]: True dat. isSinglePageApplication: true

File: src/js/components/award/idv/activity/chart/ActivityChart.jsx [hunk 4]
```diff
@@ -82,6 +82,7 @@ export default class ActivityChart extends React.Component {
 
     createBars() {
         if (!this.state.bars) return null;
+        // testing travis lighthouse
```
Comment (MaxwellKendall) [inline src/js/components/award/idv/activity/chart/ActivityChart.jsx:85]: remove this?

File: package.json [hunk 5]
```diff
@@ -12,7 +12,8 @@
         "errors": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" --quiet; exit 0;",
         "dev": "npx babel-node --no-babelrc src/js/scripts/buildSitemap.js --only 'src/js/scripts/**' && webpack --progress --config ./webpack/webpack.dev.config.js",
         "prod": "npx babel-node --no-babelrc src/js/scripts/buildSitemap.js --only 'src/js/scripts/**' && webpack --progress --config ./webpack/webpack.prod.config.js",
-        "travis": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" && ./node_modules/.bin/jest --runInBand;"
+        "travis": "./node_modules/.bin/eslint --config .eslintrc --ext .jsx,.js \"src/js/**\" && ./node_modules/.bin/jest --runInBand;",
+        "lh": "npx webpack --progress --config ./webpack/webpack.prod.config.js && lhci autorun"
```
Comment (MaxwellKendall) [inline package.json:16]: could we rename this to `light-house`?

**Review by MaxwellKendall — CHANGES_REQUESTED** [review]

**Review by MaxwellKendall — APPROVED** [review]

**Pull request merged**